examples/rapier2d_colliders.rs

```diff
@@ -109,20 +117,29 @@ fn boulders_spawn(
         return;
     };
     let sprite_image = image_assets.get(sprite_handle).unwrap();
-    let edges = Edges::from(sprite_image);
-    let coord_group = edges.multi_image_edge_translated();
-    let colliders = generate_colliders(sprite_image, ColliderType::ConvexPolyline, true);
+    let edges = Edges::try_from(sprite_image).unwrap();
+    let polygons = edges.iter();
+    let colliders = AbstractCollidersBuilder::try_from(sprite_image)
+        .unwrap()
+        .absolute()
```

Comment:
this is another instance where `polyline` (the default when omitted I believe?) isn't behaving but gravity works if you add `.convex_polyline()`